src/ring_theory/finiteness.lean

```diff
@@ -561,6 +640,13 @@ lemma comp {g : B →+* C} {f : A →+* B} (hg : g.finite_presentation) (hf : f.
   end }
 hf hg
 
+lemma of_comp_finite_type {g : B →+* C} {f : A →+* B} (hg : (g.comp f).finite_presentation)
```

Comment:
I think I would make `f` explicit. Typing one extra `_` is less annoying then typing a long `@` invocation. And chances are that `hg` and `hf` are not yet available, so `f` can't be unified, because the user wants `hg` and `hf` as new goals.